lnbits/app.py

```diff
@@ -167,6 +167,10 @@ def create_app() -> FastAPI:
     static = StaticFiles(directory=static_path)
     app.mount("/static", static, name="static")
 
+    images_path = os.path.abspath(os.path.join(settings.lnbits_data_folder, "images"))
+    os.makedirs(images_path, exist_ok=True)
+    app.mount("/library", StaticFiles(directory=images_path), name="library")
```

Comment:
nitpick: rename `library` to `img`

Reply:
dir should be created from `server.py`, like our other dirs

Reply:
https://github.com/lnbits/lnbits/blob/dev/lnbits/server.py#L41